### DISCUSSION TITLE
Add_index without needing to update the view

### DISCUSSION THREAD

**Discussion by stanleypliu** [scenic-views/scenic#381] [Q&A] [answered]
I am confused about whether or not I need a new version of my materialized view if all I'm adding is an index to a column in it. Do I still need a new version, as well as the associated migration to update it? Thanks

**Comment by derekprior** [comment, marked as answer]
You do not need to update the view just to add an index. You can use a migration with `add_index`.

**Reply by stanleypliu** [reply]
Thanks :)
